src/components/Select/utils/multiSelectHelper.ts

```diff
@@ -0,0 +1,43 @@
+import log from "loglevel";
+const logger = log.getLogger("multiselect-helper-logger");
+logger.disableAll();
+
+// 120px is the space reserved for the badge chip, the 2 buttons and some padding
+const MIN_CHIP_WIDTH = 120;
+export function calculateWidthsUntilExceed(
```

Comment:
If I'm understanding this correctly, I think a better name for this would be:
```suggestion
export function calculateCollapsedChips(
```

Reply:
ok

Reply:
I think this utility function needs a comment at the top that explains what it does because it is not obvious just from the name. It should document its overall purpose, the parameters it takes and its return value.

Reply:
code updated, resolving comment thread